### ISSUE TITLE
Updating ActiveFedora::File.mime-type

### ISSUE THREAD

**Issue by jcoyne** [projecthydra/active_fedora#890]
You must do a sparql update on ebucore:hasMimeType

**Comment by jcoyne** [comment]
Current problem:
```ruby
file = ActiveFedora::File.new
file.mime_type = 'text/xml'
file.content = 'test stuff'
file.save!

f2 = ActiveFedora::File.find(file.id)
f2.mime_type = 'text/plain'
f2.save!

f3 = ActiveFedora::File.find(file.id)
f3.mime_type
=> "text/xml"
```

**Comment by terrellt** [comment]
That's odd. Do you know why?

**Comment by jcoyne** [comment]
We set mime-type originally by sending the Content-Type headers.  This gets converted by Fedora into triples at the fcr:metadata node.  We've never built a way to update that triple.

**Comment by terrellt** [comment]
So https://github.com/projecthydra/active_fedora/tree/file_mime_types almost works, but is blocked by https://jira.duraspace.org/browse/FCREPO-1720.

**Comment by jcoyne** [comment]
FCREPO-1720 was fixed in Fedora 4.4.0. @tpendragon can that branch now be made into a PR?
